marko.json

```diff
@@ -480,6 +480,7 @@
     "@html-attributes": "expression",
     "@class": "string",
     "@style": "string",
+    "@fill": "boolean",
```

Comment:
Just spotted this. Another potential boolean trap, but I think we all realise the entire component is in need of a refactor/separation, so let's deal with it then. I will rename this to `fill-window` and update the README to make usage a little clearer in the interim.